common/Mesh-Network-Joining-Process-Comparison.md

```diff
@@ -0,0 +1,540 @@
+
+The architecture picture below illustrates the Bluetooth mesh architecture, at the bottom of the mesh architecture, there is a layer entitled as Bluetooth Low Energy. In fact, it's the Bluetooth LE stack which is required to provide fundamental wireless communications capabilities which are leveraged by the mesh architecture which sits on top of it. It should be clear that the mesh system is dependent upon the availability of the Bluetooth LE stack.   
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-bluetooth-mesh-architecture.png">  
+</div>  
+<div align="center">
+  <b> Bluetooth mesh architecture </b>
+</div>  
+<br>
+
+And the architecture of the Bluetooth Mesh stack below illustrates that the Bluetooth Mesh is almost a second stack sitting next to the Bluetooth LE stack but share the Bluetooth link layer and RF.   
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-bluetoothmesh-stack.png">  
+</div>  
+<div align="center">
+  <b> SiliconLabs Bluetooth mesh stack architecture </b>
+</div>  
+<br>
+
+## 3.2. Network topology
+Unlike the Zigbee which uses IEEE 802.15.4 MAC layer, and the MAC layer defined the mechanism for network forming and joining, the Bluetooth Mesh use the Bluetooth Low energy transfer as the fundamental wireless communication, and the provisioner will create the mesh network, and add devices to the mesh network. And the Network layer as illustrated in the Bluetooth mesh architecture defines the network PDU format, decrypts and authenticates and forwards the received incoming message to upper layer, and encrypt and authenticates and forwards outgoing message to lower layer.   
+
+Below is a typical network topology for using the smart phone as a provisioner. The Bluetooth mesh mobile application running on the smart phone will discovery and provision devices over GATT.   
+As most of the smartphones in the market at this point do not natively support Bluetooth mesh, a Bluetooth mesh stack for the phones is necessary for the phone to be able to provision, configure, and control the Bluetooth mesh nodes over the GATT bearer.   
+
+<div align="center">
+  <img src="files/CM-Mesh-Network-Comparison/joining-process-btmesh-topology-2.png">  
+</div>  
+<div align="center">
+  <b> Bluetooth mesh network topology </b>
+</div>  
+<br>
+
+## 3.3. Node type in Bluetooth Mesh network
+**Provisioner**: 
+The provisioner is the node that is capable of adding a device to a mesh network.   
+The provisioner will implement a ***Configuration Client model*** which will manage the network resources and allocate them to nodes. It manages allocation of addresses to make sure no duplicate unicast addresses are allocated, whereas a Configuration Client generates and distributes network and application keys and makes sure that devices that need to communicate with each other share proper keys for both network and access layers.   
+
+**Device**:
```

Comment:
[Mark]The title of this section is "Node type in Bluetooth Mesh network". But "A device that is not a member of a mesh network" looks conflict with the section

Reply:
Thanks, how about change it as "Member of Bluetooth Mesh Network" ?